ethereum/p2p/src/test/java/tech/pegasys/pantheon/ethereum/p2p/network/DefaultP2PNetworkTest.java

```diff
@@ -103,17 +104,19 @@ public void closeVertx() {
   @Test
   public void addingMaintainedNetworkPeerStartsConnection() {
     final DefaultP2PNetwork network = mockNetwork();
+    network.start();
     final Peer peer = mockPeer();
 
     assertThat(network.addMaintainConnectionPeer(peer)).isTrue();
 
     assertThat(network.peerMaintainConnectionList).contains(peer);
-    verify(network, times(1)).connect(peer);
+    verify(network, atLeast(1)).connect(peer);
```

Comment:
The function `peerMaintainConnectionList` is now called shortly after startup so that that we'll initiate connections to peers that were added before startup (we were previously initiating connections as peers were added in the [`RunnerBuilder`](https://github.com/PegaSysEng/pantheon/blob/e95d0681450b12df36102d16c1b99c02ea2aca2f/pantheon/src/main/java/tech/pegasys/pantheon/RunnerBuilder.java#L317)).  Update these checks to be looser in case the startup check runs in addition to the manual invocation.